### PR TITLE
RIA-4935 Decision: Appellant Timeline page - Completed actions update and 'Decision and reasons'

### PR DESCRIPTION
**Pull request by gavinslilley**
### JIRA link (if applicable) ###
[RIA-4935](https://tools.hmcts.net/jira/browse/RIA-4935)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by bradsorour — COMMENTED** [review]
@gavinslilley I do think the correct event for this is still the `sendDecisionAndReasons` event. In LR journey, only once `sendDecisionAndReasons` event has completed, then we indicate that the document is available in the documents tab. I don't believe that the Decision & Reasons documents would be available after the `startDecisionAndReasons` event has completed (as two additional events still need to happen: generateDecisionAndReasons and then sendDecisionAndReasons).

File: locale/en.json [hunk 1]
```diff
@@ -789,6 +790,17 @@
               "href": "{{ paths.common.hearingNoticeViewer }}"
             }
           ]
+        },
+        "decisionAndReasonsStarted": {
```
Comment (bradsorour) [inline locale/en.json:794]: @gavinslilley unsure if this name needs to change?

Comment (gavinslilley) [reply locale/en.json]: Yes, lets keep things consistent

**Review by bradsorour — APPROVED** [review]
LGTM. Thanks for the last minute rebase.

**Pull request merged**